File: DSA With C/Using Array/Stack.c
//WAP to implement stack using array
#include<stdio.h>
#include<stdlib.h>
#define SIZE 3
void push();//function declaration
void pop();//function declaration
void display();//function declaration//peep()   peek()
int stack[SIZE], top = -1,c;//variable declaration
void main()
{
     while(1)
     {
          printf("\n\n***** MENU *****\n1. Push\n2. Pop\n3. Exit\nEnter your choice : ");
          scanf("%d",&c);
          switch(c)
          {
               case 1: 
                    push();//func. calling
                    break;
               case 2: 
                    pop();//func. calling
                    break;
               case 3: 
                    exit(0);
               default:
                    printf("\nWrong selection!!! Try again!!!");
          }
     }
}
void push()//func. definition
{
     if(top == SIZE-1)
     {
          printf("\nStack is Full!!! Insertion is not possible!!!");
          return;               
     }
     printf("Enter the No. = ");
     scanf("%d",&stack[++top]);
     printf("\nInsertion success!!!");
     display();
}
void pop()//func. definition
{
     if(top == -1)
     {
          printf("\nStack is Empty!!! Deletion is not possible!!!");
          return;
     }
     printf("\nDeleted : %d\n", stack[top--]);
     display();
}
void display()//func. definition
{
     if (top == - 1)
     {
          printf("Stack is empty \n");
          return;
     }
     printf("Stack is : \n");
     for (int i = 0; i <= top; i++)
          printf("%d ", stack[i]);
     printf("\n");
}

File: DSA With Java/Code/using Array/Stack.java
import java.util.*;
class A{
     public int top=-1,n=5,stack[]=new int[n];
     public void push(int a)
     {
        if(top==n-1)
        {
            System.out.println("Stack Overflow!!!!!!!");
            return;
        }
        stack[++top]=a;
        System.out.println("Insertion Success ");
        dis();
     }
     public void pop()
     {
        if(top==-1)
        {
            System.out.println("Stack Underflow!!!!!!!!");
            return;
        }
        System.out.println("Deleted Element = "+stack[top--]);
        dis();
     }
     public void dis()
     {
        for(int i=0;i<=top;i++)
        {
            System.out.println(" " +stack[i]);
        }
     }
}
class Stack{
    public static void main(String[] args) {
        A kb=new A();
        int ch,a;
        Scanner ob=new Scanner(System.in);
        while(true){
        System.out.println("1. For Push \n 2. For Pop \n 3. For Exit ");
        System.out.println("Enter your Choice = ");
        ch=ob.nextInt();
        switch(ch)
        {
            case 1:
                System.out.println("Enter the Element = ");
                a=ob.nextInt();
                kb.push(a);
                break;
            case 2:
                kb.pop();
                break;
            case 3:
                System.exit(0);
            default:
                System.out.println("Vul Val Choice Dish Na vauu!!!");
        }
      }
    }
}